app/models/like.rb

```diff
@@ -1,4 +1,8 @@
 class Like < ApplicationRecord
   belongs_to :user
   belongs_to :post
+
```

Comment:
- [ ] It would be nice to add the after_save :update_likes_counter method to update the database after a like has been added 👍🏿